src/Microsoft.DocAsCode.MarkdownLite/SourceInfo.cs

```diff
@@ -34,7 +34,7 @@ public SourceInfo Copy(string markdown, int lineOffset = 0)
         private static int GetValidLineCount(string markdown)
         {
             var indexOfLastChar = markdown.Length - 1;
-            var validLineCount = 0;
+            var validLineCount = 1;
```

Comment:
Yes ,should consider about it

---
In reply to: [77107188](https://github.com/dotnet/docfx/pull/654#discussion_r77107188) [](ancestors = 77107188)